libs/account-lib/src/types/dtos/accounts.request.dto.ts

```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line max-classes-per-file
 import { ApiProperty } from '@nestjs/swagger';
 import { IsHexadecimal, IsNotEmpty } from 'class-validator';
-import { TransactionType } from '#account-lib';
+import { TransactionType } from '#account-lib/types/enums';
```

Comment:
This caused a Dependency cycle. I think this is related to the indexing problems we saw previously when importing and the services were being included when not needed.